pkg/api/types.go

```diff
@@ -754,6 +754,10 @@ type FlexVolumeSource struct {
 	// scripts.
 	// +optional
 	SecretRef *LocalObjectReference
+	// Optional: Defaults to true. SELinuxRelabel controls relabeling of the mounted volume and allows
+	// turning this off for filesystems that don't support this functionality. e.g. fuse based filesystems
+	// +optional
+	SELinuxRelabel *bool
```

Comment:
I sort of hate making users specify this - can't it just be a no-op if the driver doesn't support it?